types/vote_test.go

```diff
@@ -490,7 +490,7 @@ func TestVoteProtobuf(t *testing.T) {
 		passesValidateBasic bool
 	}{
 		{"success", vote, true, true},
-		{"fail vote validate basic", &Vote{}, true, false},
+		{"fail vote validate basic", &Vote{}, false, false},
```

Comment:
@codchen Can I ask why this test was specifically asserting that an invalid vote is construct-able from Proto?

I have changed the assertion to reject it (now that all *FromProto functions consistently call validation), and would love your eyes on it please! Thanks